src/handlers/account-deletion-processor-handler.ts

```diff
@@ -0,0 +1,62 @@
+import logger from '../commons/logger';
+import { DynamoDbService as DynamoDatabaseService } from '../services/dynamo-db-service';
+import { LOGS_PREFIX_SENSITIVE_INFO } from '../data-types/constants';
+import { AppConfigService } from '../services/app-config-service';
+import type { Context, SNSMessage, SQSEvent, SQSRecord } from 'aws-lambda';
+import { DeleteStatusUpdateSNSMessage } from '../data-types/interfaces';
+
+const appConfig = AppConfigService.getInstance();
+const ddbService = new DynamoDatabaseService(appConfig.tableName);
+
+export const handler = async (event: SQSEvent, context: Context): Promise<void> => {
+  logger.addContext(context);
+  if (!event.Records[0] || event.Records.length === 0) {
+    logger.error('The event does not contain any records.');
+    return;
+  }
+
+  const updateRecordsByIdPromises = event.Records.map((record) => {
+    const userId = getUserId(record);
+    return userId ? updateDeleteStatusId(userId) : undefined;
+  }).filter((prom) => prom !== undefined);
+  await Promise.all(updateRecordsByIdPromises);
+};
+
+function getUserId(record: SQSRecord) {
+  let messageBody: SNSMessage;
+  let message: DeleteStatusUpdateSNSMessage;
+  try {
+    messageBody = JSON.parse(record.body);
+    message = JSON.parse(messageBody.Message);
+  } catch {
+    logger.error('The SQS message can not be accessed.');
```

Comment:
minor thing: can you please change this error message to something related to the fact that the message cannot per parsed? Saying access may make people think of a permission issue